packages/clarity-js/src/performance/observer.ts

```diff
@@ -83,9 +88,37 @@ function process(entries: PerformanceEntryList): void {
     }
 }
 
+function calculateCls(entry: PerformanceEntry): void {
+    if(entry["hadRecentInput"]) { return; }
+    
+    const firstSessionEntry = sessionEntries[0];
+    const lastSessionEntry = sessionEntries[sessionEntries.length - 1];
+
+    // If the entry occurred less than 1 second after the previous entry
+    // and less than 5 seconds after the first entry in the session,
+    // include the entry in the current session. Otherwise, start a new
+    // session.
+    // For more info: https://web.dev/articles/cls
+    if (
+        sessionValue &&
+        lastSessionEntry && (entry.startTime - lastSessionEntry.startTime < 1000) &&
+        firstSessionEntry && (entry.startTime - firstSessionEntry.startTime < 5000)
+    ) {
+        sessionValue += entry["value"];
+        sessionEntries.push(entry);
+    } else {
+        sessionValue = entry["value"];
```

Comment:
>sessionValue [](http://example.com/codeflow?start=8&length=12)

we used to multiply by 1000, how are we handling the transition for the backend here?